content/sips/sip-182.md

```diff
@@ -0,0 +1,161 @@
+
+### Rationale
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+This SIP makes it easier to deploy `Wrapper` contracts that can support more assets in the future and will help expand the total supply of synths.
+
+### Technical Specification
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
+
+#### WrapperFactory
+
+A new Synthetix core contract is created, `WrapperFactory`, which by means of the factory pattern deploys `Wrapper` instances using `createWrapper`. The `WrapperFactory` assumes functionality previously exposed by `EtherWrapper`.
+
+Specifically, `WrapperFactory` contains the following functions:
+- `createWrapper(IERC20 token, bytes32 currencyKey, bytes32 synthContractName)`
+  - Allows `owner` to create `Wrapper` contracts. `token` and `currencyKey` indicate which token and synth should be wrapped, and `synthContractName` is included because concatenation of `bytes32` strings within solidity is difficult, and should match the `AddressResolver` name of the contract for `currencyKey`.
+- `isWrapper(address possibleWrapper) returns (bool)`
+  - Allows Synthetix system components to verify a valid Wrapper contract by passing contract address. Returns boolean, indicating if its a Wrapper contract or not
+- `distributeFees()`
+  - Assumes functionality of `EtherWrapper.distributeFees`, sending fees to the fee pool and informing of status
+- `totalIssuedSynths() returns (uint)`
+  - TBD
+
+#### Wrapper
+
+The Wrapper created by the factory should be functionally equivalent to [SIP-112](sip-112.md), with the following exceptions:
+- Constructor accepts IERC20 token address `token` of source token, and currency key `currencyKey` of target synth
+- `sETHIssued` renamed to `synthIssued`
+- `maxETH` renamed to `maxTokenAmount`
+- `SystemSettings` values have additional `currencykey` argument to match
+- Upon `_mint` or `_burn`, equivalent value sUSD is minted into the `WrapperFactory` contract instead of withholding `sETH`
+- `distributeFees` removed
+
+`EtherWrapper` and `NativeEtherWrapper` are deprecated.
+
+#### DebtCache
+
+This class will be modified TBD
```

Comment:
This is pretty critical to the SIP IMO. Doesn't require deep technical explanation but should be discussed as it's a critical part of external collateral and calculating debt.

Reply:
yeah i approved the PR but definitely should update this before presenting to the council